app/experimenter/nimbus-ui/src/components/Summary/EndExperiment/index.tsx

```diff
@@ -53,6 +54,7 @@ const EndExperiment = ({
         <Button
           variant="primary"
           onClick={toggleShowEndConfirmation}
+          className="EndExperimentStartButton"
```

Comment:
(same for L35 above) I think our current convention is kebab-lower-case for class names.
For example, `className="end-experiment-start-button"` and `className="end-experiment-confirm-button"`. Unless you want to try finding a way to reuse the `data-testid` attribute (possibly like `[data-testid='end-experiment-start']`).

Reply:
Good point, I'll change this.